### ISSUE TITLE
[Bug] Digital signature bootstrapper error

### ISSUE THREAD

**Issue by pvx** [oleg-shilo/wixsharp#1407]
The error messsage:
```
EXEC : SignTool error : Invalid command: -ib [C:\GoAgent\pipelines\Prod.SignedClient\client\src\
Installer\Installer.csproj]
```
Log before error:
```
Execute:
"C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\x86\signtool.exe" -ib "C:\GoAgent\pipelines\Prod.SignedClient\client\src\RELEASES\Client.exe" -o "C:\Users\Acc\AppData\Local\Temp\tmpA324.tmp"
```
![image](https://github.com/oleg-shilo/wixsharp/assets/1254103/2df578ed-e55b-4da8-9f82-14ad9108616c)

When I fixed the code as above it was build without any errors and installer has been signed.

**Comment by NJullienSweet** [comment]
This is related to #1289

You are having this issue because `insignia` has been removed from WiX in v4. If I understood correctly, the new way which is not yet implemented in WixSharp is to use the following commands:
- `wix burn detach [options] original.exe -engine engine.exe` to extract the engine from your Bundle EXE
- `signtool sign [options] engine.exe` to sign the extracted engine 
- `wix burn reattach [options] original.exe -engine signed.exe -o final.exe` to put back the signed engine into your Bundle EXE
- `signtool sign [options] final.exe` to sign the reassembled Bundle EXE

See https://wixtoolset.org/docs/tools/wixexe/#burnsigning

I ended up copying the `DigitalSignatureBootstrapper` class and changing its calls to use my own `DigitalySignBootstrapperEngine` method to use these commands.

I'm considering opening a PR to WixSharp to properly implement this, if @oleg-shilo agrees.
